useful-scripts/scripts/copyTrello.swift

```diff
@@ -0,0 +1,218 @@
+import Foundation
+import Alamofire
+
+extension String {
+
+    func getReferencedURL() -> String {
+        do {
+            let URLRegex = "^((http[s]?|ftp):\\/)?\\/?([^:\\/\\s]+)((\\/\\w+)*\\/)([\\w\\-\\.]+[^#?\\s]+)(.*)?(#[\\w\\-]+)?$"
```

Comment:
I'd extract this to a static property in this class

Reply:
Done